control-plane/experiencelab.sh

```diff
@@ -582,6 +582,9 @@ function CreateFirstUser() {
   Verify $5 'CreateFirstUser-ERROR: Argument (DATA_PARTITION) not received'
   Verify $6 'CreateFirstUser-ERROR: Argument (FIRST_USER) not received'
 
+  echo "  Waiting 10 min for the instance to come up"
+  sleep 600
```

Comment:
Can we change the wait to be more directly tied to the completion of creating the instance? That way if we move something else before creating the user we don't have to remember to move the sleep

Reply:
updated